cip-smart-tx-signing/README.md

```diff
@@ -0,0 +1,82 @@
+---
+CIP: ?
+Title: Wallet Transaction Caching
+Status: Active
+Category: Wallets
+Authors:
+    - Josh Marchand <josh@securitybot.info>
+Implementors: N/A
+Discussions:
+ - N/A
+Created: 2023-12-22
+License: CC-BY-4.0
+---
+
+# CIP-? Wallet Transaction Caching
+
```

Comment:
```suggestion
```
CIP-0001 says "no H1 title` generally since this is constructed from the header in other contexts.